specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json

```diff
@@ -379,6 +379,14 @@
             "$ref": "./network.json#/definitions/SubResource"
           },
           "description": "A collection of references to application gateway path rules."
+        },
+        "applicationGatewayForContainers": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGatewayForContainersReferenceDefinition"
```

Comment:
Instead of this, why not just use ApplicationGatewayForContainer like we do for CustomRule?

Reply:
Keeping the name similar to the class name in NRP
https://msazure.visualstudio.com/One/_git/Networking-nrp?path=/src/sources/Frontend/FrontendContract/Csm.Public/ApplicationGatewayForContainersReference.cs